azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -663,3 +663,16 @@ def invoke_device_module_method(self, device_id, module_id, direct_method_reques
         return self.protocol.device_method.invoke_module_method(
             device_id, module_id, direct_method_request
         )
+
+    def send_c2d_message(self, device_id, message):
+        """Send a C2D mesage to a IoTHub Device.
+
+        :param str device_id: The name (Id) of the device.
+        :param str message: The message that is to be delievered to the device.
+
+        :raises: Exception if the Send command is not able to send the message
+        """
+        svc_client = iothub_amqp_client(
```

Comment:
>svc_client = iothub_amqp_client( [](start = 8, length = 32)

Are we creating the amqp_client every time when we send a message? I think the amqp_client should be a member of the class and should be initialized alogn with the protocol client.